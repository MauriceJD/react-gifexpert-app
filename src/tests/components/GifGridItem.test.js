import React from 'react';
import {shallow} from 'enzyme';
import GifGridItem from '../../components/GifGridItem';


describe('Pruebas en el archivo GifGridItem.js', () => {
    test('debe mostrar el componente correctamente', () => {

        const title = 'Hola que tal saludos';
        const url = 'https://localhost/algo.jpg'

        const wrapper = shallow(<GifGridItem title={title} url={url}/>)
        expect(wrapper).toMatchSnapshot()
    })
    
})


