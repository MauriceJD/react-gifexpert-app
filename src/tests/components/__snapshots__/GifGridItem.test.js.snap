// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el archivo GifGridItem.js debe mostrar el componente correctamente 1`] = `
<div
  className="card animate__animated animate__bounce"
>
  <img
    alt="Hola que tal saludos"
    src="https://localhost/algo.jpg"
  />
  <p>
    Hola que tal saludos
  </p>
</div>
`;
